lib/Service/AutoProvisioningService.php

```diff
@@ -57,21 +58,27 @@ class AutoProvisioningService {
 	 * @var IClientService
 	 */
 	private $clientService;
+	/**
+	 * @var Client
+	 */
+	private $client;
 
 	public function __construct(
 		IUserManager $userManager,
 		IGroupManager $groupManager,
 		IAvatarManager $avatarManager,
 		IClientService $clientService,
 		ILogger $logger,
-		IConfig $config
+		IConfig $config,
+		Client $client
 	) {
 		$this->userManager = $userManager;
 		$this->groupManager = $groupManager;
 		$this->avatarManager = $avatarManager;
 		$this->logger = $logger;
 		$this->config = $config;
```

Comment:
since the field config is no longer needed please remove it - TXH a lot! :+1:

Reply:
Done in commit 'remove unused config var'